src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -388,48 +328,39 @@ private void DestroyAccounts(TransactionSubstate substate)
         {
             foreach (var toBeDestroyed in substate.DestroyList)
             {
-                if (_logger.IsEnabled(LogEventLevel.Verbose))
-                {
-                    _logger.Verbose($"Destroying account {toBeDestroyed}");
-                }
-                
+                if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose($"Destroying account {toBeDestroyed}");
+
                 _stateProvider.DeleteAccount(toBeDestroyed);
             }
         }
-        
-        private void DeployCode(ExecutionEnvironment env, TransactionSubstate substate, ref ulong unspentGas, IReleaseSpec spec)
+
+        private void DeployCode(ExecutionEnvironment env,
+            TransactionSubstate substate,
+            ref ulong unspentGas,
+            IReleaseSpec spec)
         {
             var codeDepositGasCost = (ulong) CodeDepositHandler.CalculateCost(substate.Output.Length, spec);
-            if (unspentGas < codeDepositGasCost && spec.IsEip2Enabled)
-            {
-                throw new OutOfGasException();
-            }
+            if (unspentGas < codeDepositGasCost && spec.IsEip2Enabled) throw new OutOfGasException();
+
+            if (unspentGas < codeDepositGasCost) return;
 
-            if (unspentGas < codeDepositGasCost)
-            {
-                return;
-            }
-            
             var codeHash = _stateProvider.UpdateCode(substate.Output);
             _stateProvider.UpdateCodeHash(env.CodeSource, codeHash, spec);
             unspentGas -= codeDepositGasCost;
         }
 
-        private (Address sender, Address recipient) ExtractSenderAndRecipient(ContractEntry entry)
+        private (Address sender, Address recipient) ExtractSenderAndRecipient(PublicEntry entry)
         {
             var sender = GetAccountAddress(entry.Base.SenderPublicKey);
             var recipient = entry.TargetContract == null
                 ? GetAccountAddress(entry.Base.ReceiverPublicKey)
                 : new Address(entry.TargetContract);
-            if (entry.IsValidDeploymentEntry)
-            {
-                recipient = Address.OfContract(sender, _stateProvider.GetNonce(sender));
-            }
+            if (entry.IsValidDeploymentEntry) recipient = Address.OfContract(sender, _stateProvider.GetNonce(sender));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)